app/modules/usb/migrations/2018_07_30_191408_usb_add_product_id.php

```diff
@@ -0,0 +1,29 @@
+<?php
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class UsbAddProductId extends Migration
+{
+    private $tableName = 'usb';
+
+    public function up()
+    {
+        $capsule = new Capsule();
+        $capsule::schema()->table($this->tableName, function (Blueprint $table) {
+          $table->string('product_id')->nullable();
+
+          $table->index('product_id', 'product_id_index');
```

Comment:
Please prepend the index name with 'usb_' or don't specify the index name and let eloquent generate the name.

Reply:
I've removed the name of the index, thanks!